lib/Twig/TemplateWrapper.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2016 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Exposes a template to userland.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class Twig_TemplateWrapper
```

Comment:
I would have loved to use `Twig_Template`, but that's not possible. So, I used a descriptive name instead. But I agree that the name is not ideal.